docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -59,7 +58,7 @@ public static void Example()
                 IDataView testDataset = trainTestData.TestSet;
 
                 var pipeline = mlContext.Model.ImageClassification(
-                    "ImagePath", "Label",
+                    "ImageVBuf", "Label",
```

Comment:
>ImageVBuf [](start = 21, length = 9)

Please rename "ImageVBuf" to "Image" here and please call "VBuffer" as "VBuffer" and not "VBuf" :-) I don't think we use "Buf" for "buffer" anywhere in this codebase.

Reply:
Done. Changed it to Image.